src/test/java/uk/gov/digital/ho/hocs/client/audit/AuditClientTest.java

```diff
@@ -46,62 +57,63 @@
     private final ClientContext clientContext = new ClientContext();
```

Comment:
You didn't do the work, but I'm not sure why we've done this differently in this project to all the others, have you observed the headers being populated in the messages?

Reply:
They are being populated when running locally...